docs/user/kubernetes-auth.md

```diff
@@ -0,0 +1,25 @@
+# Authorization with kubernetes
+
+Flight control kubernetes authorization uses Kubernetes RBAC to control authorization for flight control API endpoints.
+Some helm configuration values are needed to be modified (see [values.yaml](https://github.com/flightctl/flightctl/blob/main/deploy/helm/flightctl/values.yaml#L34-L41))
```

Comment:
Linking to specific lines is brittle. We'll probably want to link to a (to be created) "Installing Flight Control on Kubernetes" section later.